Common/buffered_lane.h

```diff
@@ -109,8 +110,9 @@ class lane_read_buffer {
   lane_read_buffer(const lane_read_buffer&) = delete;
   lane_read_buffer& operator=(const lane_read_buffer&) = delete;
 
-  Tp* _buffer;
-  size_t _buffer_size, _buffer_pos, _buffer_fill_count;
+  aocommon::UVector<Tp> _buffer;
```

Comment:
Good change I think, however is this tested? Also, since this is in aocommon, we should push it back to where it came from.

Reply:
The unit test coverage report says lines 104-106 (in the read() function) are not covered. Although I didn't touch those lines, it would be nice to test if that read() function still works, when lane_->read(...) doesn't return 0. Since that read() function is only used in Interpolate.cc, we'd have to extend tInterpolate.cc (which only has 1 test now) or add a new unit test for buffered_lane.h.

Regarding the repository: This file is in DP3/Common and not in the aocommon subrepo.